source/index.html

```diff
@@ -1,12 +1,185 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: {{projectTitle}}</title>
+    <meta charset="utf-8" />
+    <title>CAT ENERGY</title>
   </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
+  <body class="page__body">
+    <header class="main-header">
+      <a class="main-header__logo" href="#">
+        <img class="main-header__logo-image" src="#" alt="#" />
+      </a>
+      <a class="main-header__title" href="#">
+        <img class="main-header__title-logo" src="#" alt="#" />
+      </a>
+      <nav class="main-nav">
+        <button class="main-nav__toggle">
+          <span class="visually-hidden">Открыть меню.</span>
+        </button>
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list">
+            <li class="main-nav__item main-nav__item--current">
+              <a href="#">главная</a>
+            </li>
+            <li class="main-nav__item">
+              <a href="catalog.html">каталог продукции</a>
+            </li>
+            <li class="main-nav__item">
+              <a href="form.html">подбор программы</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main class="main-container">
+      <section class="functional-food">
+        <div class="functional-food__wrapper">
+          <h1 class="functional-food__title">
+            Функциональное питание для котов
+          </h1>
+          <p class="functional-food__text">Занялся собой? Займись котом!</p>
+          <img class="functional-food__image" src="#" alt="#" />
+          <button class="functional-food__button button">
+            подобрать программу
+          </button>
+        </div>
+      </section>
+      <section class="food-category">
+        <h2 class="visually-hidden">
+          Выбор правильного фунцкионального питания.
+        </h2>
+        <div class="food-category__wrapper">
+          <div class="food-category__card card">
```

Comment:
```suggestion
          <div class="card">
```
я бы так оставил и дальше описывал блок уже карточки